ifrs17-template/Initialization/InitSystemorphUtils.ipynb

```diff
@@ -0,0 +1,154 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "6vRP2T2AiEq5rmfljOKoSQ",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> Utility Methods for Systemorph Initialization </span> </p>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "using Systemorph.Vertex.DataSource;"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Files by Data Node"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "static readonly Dictionary<string,(string FileName, string Format)[]> DataNodeFiles = ",
+                "\n                                                    new Dictionary<string,(string FileName, string Format)[]>(){{\"ES\", new (string, string)[]{(\"DataNodes/DataNodes_ES.csv\", ImportFormats.DataNode), ",
```

Comment:
go to new line each new company (go to new line with first ES kvp

Reply:
done